mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -872,9 +875,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
         //menu items
         switch (item.getItemId()) {
             case R.id.mainmenu_openhab_preferences:
-                Intent settingsIntent = new Intent(this.getApplicationContext(), OpenHABPreferencesActivity.class);
-                startActivityForResult(settingsIntent, SETTINGS_REQUEST_CODE);
-                Util.overridePendingTransition(this, false);
+                showMessageToUser(
+                        getString(R.string.info_settings_moved),
+                        Constants.MESSAGES.TOAST,
+                        Constants.MESSAGES.LOGLEVEL.ALWAYS
+                );
+                openPreferences();
```

Comment:
I thought this menu item should be killed?

Reply:
We probably haven't agreed on that, but I think it's ok to directly kill it :)